components/top-nav/src/atoms/top-nav-inner/index.js

```diff
@@ -9,10 +9,10 @@ const TopNavInner = styled('div')({
   flexDirection: 'column',
   width: `calc(100% - ${SPACING.SCALE_3})`,
   maxWidth: '960px',
```

Comment:
should we get rid of this?

Reply:
This allows flex column stacking on small screen sizes.